dev-packages/electron/package.json

```diff
@@ -14,6 +14,12 @@
     "url": "https://github.com/eclipse-theia/theia/issues"
   },
   "homepage": "https://github.com/eclipse-theia/theia",
+  "files": [
+    "native",
```

Comment:
The file causing issues is under this folder and still present in the resulting `.tar.gz` archive. Do we need to exclude it from here or create a `.npmignore` file?

Reply:
> create a .npmignore file

I've pushed a small commit on top that does that

Reply:
there were additional test commits I did not intend to push - corrected with force-push.

Reply:
That did not work - `native/build` is still present in the resulting archive. My guess is that "files" has precedence (.npmignore on its own worked). Feel free to get rid of my extra commit if you have another idea that might work.

Reply:
Ahh I forgot that in this case it would build into `native/build`, usually it's at the package's root...